cernopendata/modules/theme/assets/semantic-ui/js/search/components/FacetItem.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * -*- coding: utf-8 -*-
+ *
+ * This file is part of CERN Open Data Portal.
+ * Copyright (C) 2021 CERN.
+ *
+ * CERN Open Data Portal is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * CERN Open Data Portal is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with CERN Open Data Portal; if not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+ * MA 02111-1307, USA.
+ *
+ * In applying this license, CERN does not
+ * waive the privileges and immunities granted to it by virtue of its status
+ * as an Intergovernmental Organization or submit itself to any jurisdiction.
+ */
+
+import React from "react";
+import { List, Checkbox } from "semantic-ui-react";
+
+const FacetItem = (props) => {
+  const {
+    bucket,
+    isSelected,
+    onFilterClicked,
+    getChildAggCmps,
+    keyField,
+  } = props;
+  const label = bucket.label ? (
+    bucket.label
+  ) : (
+    <label>
+      {keyField}{" "}
+      <span className="facet-item-count">
+        ({bucket.doc_count.toLocaleString("en-US")})
+      </span>
```

Comment:
It's a pity that we have to override the whole component only because of separating the count in a different HTML element, but I don't see any other easy solution for this... 😕

Reply:
Indeed, I also spent some time trying to find another way, but with no luck